ios/RNCPagerView.m

```diff
@@ -47,6 +48,15 @@ - (void)didMoveToWindow {
     }
 }
 
+- (void)layoutSubviews {
+    [super layoutSubviews];
+    if (!_scrolledToInitialPage) {
+        _scrolledToInitialPage = YES;
+        [self goTo:self.initialPage animated:false];
+    }
+}
+
+
```

Comment:
Looks like bug causing `initialPage` sometimes not to be set correctly can be easily fixed by adding check for `contentSize` like: 
```objc
-(BOOL)isScrollViewContentSizePositive {
    return [self isHorizontal] ? _scrollView.contentSize.width > 0 : _scrollView.contentSize.height > 0;
}
```
and adding it to

```objc
-(void)layoutSubviews {
     [super layoutSubviews];
     if (self.isScrollViewContentSizePositive && !_scrolledToInitialPage) {
         _scrolledToInitialPage = YES;
         [self goTo:self.initialPage animated:false];
     }
 }
 ```
 
 Could you please add it?